### ISSUE TITLE
Importing a tree from URL loses mastery information

### ISSUE THREAD

**Issue by strey1998** [PathOfBuildingCommunity/PathOfBuilding#3373]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

After exporting a tree to URL, then importing onto a new tree, I expect that the selected mastery nodes will retain their selections.

### What is the actual behaviour/value?

The allocated mastery nodes on the imported tree are allocated, but have no effect.

### How to reproduce the issue

1. Export a tree with at least one allocated mastery node to URL.
2. Import onto a fresh tree.

### Build code

```shell
eNrtW1t32rgWfu78Ci-eSUCWfOsiM4skJKUrpEyctnOeuoQtwI2wGFtA6Vnnv49kO4mTctmoc2bNA3mg4Hxb-6qtb4u089u3GbeWLMsTkZ410Gm7YbE0EnGSTs4aH--vTvzGb7_-0hlSOf0wPl8kXP_m11_edIr3FmdLxpVcw5I0mzD56XEl_EWtNKepnDKRDuhXkV2L-KxxK1LWsEY0jRP5-CniNM9v6YydNcJICTcsmkcsjS-en5fAGU3SUEQPTF5nYjEv1C4TthqIWGH6g-GHu_ua0iStK1U2v-kMOV2zLJRUWrl6OWt0let0wi7pTL2q1ShfMB2GoE08F1WvjdZm4XDOWFwTIo79z0KHGeuNxyySyZJdZIm8mNI0enajvU3uUOxgwWUy5wnLanY52yTe_bA4am9d_l5Iyi-H4RPWPvVcx0PE9vUr_v_ICbnf8c-JnJ5zlQsDLVq2P0kTyQyFhyLJRfoT_tVFt-dWzEZJauTho2h3OTlUdEBTeiFyQApukjGDIXshDHentjkMqa0cskw1Igk39iCBUD5vJ3u7xezPOnAr7pJ9ewJhZ8dqdeDW1fqphNlWB-6wbSlk0dkhQXxCubs6b_RWY_tp9NxlnF2rfkwzlrNsWeut29d_KVBlFdLOtOAdmzCgqzeMRdNrdazdUclgNfmEcto7o6Ox9ejsXHRDcBygwAHB0YKvgnMa7AIfGJ7PNIsBrSJl2WQdThPGAegilnURUEzrAsByqIsc6HdvSfP67kLubm9KOKw4mDoBlUDMXh3pzt8nMczEV81h-GFi3WwmFhkwgyUY5PJwus6TSB2iBSe8Y_EigvWuJyo2EEs2U3uiYHKKnNZIU3urO-dcMVsoKVMrc36QRFdKGj1cinjCDlJyuES4mM9Vc9AlBpW7SjIV6DypnZonLgD9QbH3CzqHEB29D6EKntFgBTfJZCpTNRrBtbwSgfsypSI_wJlnOFjF1YJzEIEMH5KXSElHnL212t_Q2MH2aLx9l4mVMmaqh8YcsCdr6AGtEZ2tHmQs_b4Gr_8CDlLQS-NFpqsbrOO1xI9qOq1iqNbv-rO5yGTx8ILyKC-W7KfzhbTSYiLOdeS_pIvZSA9l5b96IG79gJwlefRltBiP9ZjcUMZkxWzfu7rqXdz3P_UqkZAVbc6KBOd0njM1N6cJb1hJXKU5VI09kgC0mv-qeXo_Vh3ZUZao8O2H6qENYGoxPe_H6bEWAFN5oJxBQ3S_njOd8RwWpao97scWYygAV06NELeK2RQSdBbRNQD3xFgA2B5n3YTrAxISp4Eq3_JkhaDVMZclo4UEVVQxbAAs0Kwb4FadV4IqHwKrWA5AfUUC9yOrswNSdopgAFN0ycZMFTNoIxfN4Z4-sLRqe48trlPsotzKVe-7ZrP8fK3Olisd0Fd3FS8A6tTJFqqzxWxMF1w__31BeSLXuhnXnt6Ud5WFHflUrDRRKdfR-zZXsb65KX_T5bJaQis5a4wpz8tK6dxnjFm0oHmaWKrpuLxZ1B_q95b9uNA-o7lk2boq4LPGf7FvY6dJPJfY_1N9W7Vk9RRjhzhNFNi-1ywBLvI9p-n4PsZNEmAvaNrEJbiJHERI0_ZtGzUD5Df1VQtpWFJZVbt6VQNAeataWqENfNPpxWrLx7daY6t88vHupnjzZirlPH_baq1Wq9M5lVMxZt8UHT-NxKw1V8soZ0-KA-dEK2p11c-5fukP3oeIffr85fr76sPt5PfvuRh9bV_y2QkrFLQeNXTKa9u8VFd9KrzXFiLXRr6qDclmpcGtLTiE1FS6H6eChwkERxwfA3C2HQSQ9YjvegFkPYwQABaQNiQq2HVB3mJCYN6iwIesZ5MA4gXGngNRi5ELM8_1bAeAcxwUQIqFBG0fgkMYeRCcF7ggta7v2yCc8gMUZhdD0kHapN0GpaMNUYtsF2GQGzgAZVdViwvJbuD4kCpAJADpDbzAg5jnEw-i1g48ZIOKGQUuqKPhtgPC2b4P6mgeAfUWHPgBSK_rQvxAno1gHRLcgzzYiUBAeomqZ0i5OEg1SYi_bTUFQ3A-waCyVwZ6oKbm-rATy3NBftjIBXUXRVsgfhCCXFBd2RiU3raHHdBRGYCapBsgUI-0vU3V0mnV2I76oJjhM0WUieSsmCsV9dTX7fs54xNNLMkfhCaWhPLfQhMvc_GCJpLB-yNNPNLEI0080sQjTTzSxCNNPNLEI02saGKnpW8Y9ZtbIVnxO_3w8UNx__gpYSsrZzSLpqHMNEH8LsTsP7rvnLZtxRF9B6sfUl5mvmNUDui8uvjUyOoiVBtf3INKKi8TxTaz4gb38TpVI_84a5wQ2z5VpAA5uPyzgk5fOZJXd6D6fchk8aeYi5yFLBJp_JnRuUiLx1pn-Y1UCdwKKm6GUcVBQy4ev70qYRYKV3RudUfrPKfcqgK8IZebRF-L2TCx3ahzIWRuaNAmX_YYdc64PFDkSjH7BwuDTLI3mYTNwgsLgv03iREzI10zba55NE3qaU8G3jE-Y_JQRRsqyTXRs6f8rrlY6q_FD18Ym-8iZKKPmNUCMqs8sj89kMAeuuym8jJ0HNR3HAN7DHog3qcmXlvlV6gHpm9zlI3qyzERAm1JSCKM_HSMw2ob5N0120nYrHzx4VVGzNu-QU0j4-BjSEEQCMjA7J8gP8BMmmfBNahK8zT8RLlAW7KWBe4aONQgcndJOoExgYMbxdbwGrYLeD5N3MGAOCHzNBDjaDkm3kD2i3mR7zGpO1vwDWdbp1XNkMWUXAyhxZ9HinScTBSi03r9nxT_Ahi5seM=
```

**Comment by zao** [comment]
Tree exports via "Export Tree" on the bottom bar are still v4 while current format is v6, there hasn't been much need to refresh in the past as the website didn't do meaningful stuff with cluster jewels which were the primary feature of v5.

**Comment by LocalIdentity** [comment]
Fixed in 2.12.0
